docs/Actions.md

```diff
@@ -641,6 +641,8 @@ The uploaded `version.json` file provides an easy way for apps to poll if a new
 }
 ```
 
+This action currently supports [aws-sdk v2](https://github.com/aws/aws-sdk-ruby)
```

Comment:
Ideally the user doesn't have to know anything about this. Right now we don't seem to have a dependency to the `aws-sdk`, right? 

How about we verify the `aws-sdk` version if we need it (in the `s3` action) to show the user to add the SDK to their `Gemfile`?